README.md

```diff
@@ -18,6 +18,10 @@ Roadmap:
 - OpenID Connect implementation
 - Support more DID types
 - Support non self-signed credentials
+- Zero Knowledge Proofs
+- Selective Disclosure
```

Comment:
this is already a feature, user can choose not to provide a name thats requested for example. user only has to disclose what they need to sign in (eg: email). ZKP is a way to do that for existing credentials. but id say to avoid user confusion that this feature is already "done" and can be enhanced with zkp